content/en/docs/instrumentation/erlang/getting-started.md

```diff
@@ -7,31 +7,7 @@ Welcome to the OpenTelemetry for Erlang/Elixir getting started guide! This guide
 will walk you through the basic steps in installing, configuring, and exporting
```

Comment:
Since the installation section is gone it might make sense to rework the introductory paragraph to not mention "installing"?